dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/publish/servers/com.ibm.ws.webcontainer.security.fat.basicauth.audit/server.xml

```diff
@@ -12,7 +12,6 @@
 
 	<featureManager>
 		<feature>jsp-2.2</feature>
-		<feature>servlet-3.1</feature>
```

Comment:
Not sure why this was removed. Can you explain?

Reply:
I think I took it out earlier, put it back, test still passed.